build/charts/theia/values.yaml

```
@@ -109,6 +109,9 @@ clickhouse:
         repository: "projects.registry.vmware.com/antrea/theia-zookeeper"
         pullPolicy: "IfNotPresent"
         tag: "3.8.0"
+      # Enable securitycontextconstraints
+      # set to false for openshift
+      scc: true 
```

I think the name `scc` and the comments are not clear enough. I would suggest some updates like the follows. Please keep the `--` in the comments so that it can be included to when generating the documents. 

```suggestion
      # -- Determine whether to run ZooKeeper as a non-privileged user.
      # Please set this value to false when running on OpenShift.
      securityContext:
        enable: true
```

---

Instead of making this Openshift specific I believe it's better to make the securityContext configurable.